traffic_portal/app/src/modules/public/sso/sso.tpl.html

```diff
@@ -0,0 +1,20 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div id="ssoContainer"/>
```

Comment:
Same as above RE: terminating newlines

also, as far as I can tell, this is never styled or accessed anywhere, so could it just have a blank template instead of this file? Or maybe I'm wrong and Angular will complain about blank templates. In that case, I still don't think this `id` is ever used for anything.

Reply:
Also, according to [the HTML spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-div-element) the DIV element may not omit a closing tag.

Reply:
done.  you were right, i didnt need it at all and could just use a blank template.